app/models/manageiq/providers/hawkular/middleware_manager/refresh_parser.rb

```diff
@@ -111,6 +124,17 @@ def process_server_entity(server, entity)
         end
       end
 
+      def process_availability(availability)
+        case
+        when availability.nil?, availability['value'] == 'unknown'
+          _('Unknown')
```

Comment:
I agree with you on that part. Not translating on backend would benefit the clients for using whatever language they want or even change the language at runtime.
If that's how is done in MiQ I'll remove the translation call.